include/pmacc/PMaccConfig.cmake

```diff
@@ -141,6 +141,11 @@ if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE) AND ALPAKA_ACC_GPU_CUDA_ONLY_MODE)
     message(WARNING "ALPAKA_ACC_GPU_CUDA_ONLY_MODE is set to OFF because cuda backend is not activated")
 endif()
 
+# includes and definitions are already added in cupla's & alpaka's
```

Comment:
you are right, and alpaka does it correctly!

ideally we would not need a wrapper at all but only a property set.
Shouldn't this be possible with CUDA language support in CMake 3.8+. Or should we still hide the language property from the alpaka user and only talk to them in form of alpaka backends?